Gemfile

```diff
@@ -4,7 +4,8 @@ eval_gemfile(File.expand_path("gems/pending/Gemfile", __dir__))
 # VMDB specific gems
 #
 
-gem "rails",                           "~>5.0.x"
+gem "rails",                           :github => "rails/rails", :branch => "master"
```

Comment:
I'm fine with this, until rc1 is cut (which will also be when 5-0-stable is branched)